test/base/testing_brave_browser_process.cc

```diff
@@ -1,7 +1,7 @@
-/* Copyright (c) 2021 The Brave Software Team. Distributed under the MPL2
- * license. This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
since there are no other changes please revert these back to the original so we don't have unrelated commits here